.github/workflows/conda.yml

```diff
@@ -0,0 +1,56 @@
+name: Testing with Conda
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: goanpeca/setup-miniconda@v1
+        with:
+          activate-environment: mantidimaging
+          environment-file: environment.yml
+          python-version: 3.7
+          auto-activate-base: false
+
+      - name: Other dependencies
+        shell: bash -l {0}
+        run: |
+          sudo apt update -y
+          sudo apt-get install -y xvfb libxcb* libxkb*
+
+      - name: Yapf
+        shell: bash -l {0}
+        run: |
+          yapf --parallel --diff --recursive .
+
+      - name: Flake8
+        shell: bash -l {0}
+        run: |
+          python -m flake8
+
+      - name: mypy
+        shell: bash -l {0}
+        run: |
+          mypy --ignore-missing-imports mantidimaging
+
+      - name: pytest
+        shell: bash -l {0}
+        run: |
+          xvfb-run --auto-servernum python -m pytest
+
+      - name: publish 'unstable' package
+        if: github.ref == 'refs/heads/master'
```

Comment:
Can you change this to `refs/heads/update_github_actions_versions` temporarily to see if a package publishes successfully? If it doesn't we'll have to fix it on master